packages/reports/tests/acceptance/navi-report/navi-column-config-test.js

```diff
@@ -205,6 +210,684 @@ module('Acceptance | Navi Report | Column Config', function(hooks) {
+    await clickItem('metric', 'Platform Revenue');
+    await animationsSettled();
+
+    await click('.navi-column-config-item__name[title="Platform Revenue (USD)"]');
+    await click('.navi-column-config-base__filter-icon');
+
+    assert.deepEqual(
+      findAll('.filter-builder__subject').map(el => el.textContent.trim()),
+      ['Date Time (Day)', 'Platform Revenue (USD)'],
+      'Metric filter is added'
+    );
+
+    await click(findAll('.navi-column-config-item__name[title="Platform Revenue (USD)"]')[1]);
+
+    assert.dom('.navi-column-config-base__filter-icon--active').exists({ count: 2 }, 'Both filter icons are active');
+
+    await selectChoose(findAll('.navi-column-config-metric__parameter-trigger')[1], 'Dollars (CAD)');
+
+    assert.deepEqual(
+      getColumns(),
+      ['Date Time (Day)', 'Platform Revenue (USD)', 'Platform Revenue (CAD)'],
+      'The second metric was updated'
+    );
+
+    assert
+      .dom('.navi-column-config-base__filter-icon--active')
+      .exists({ count: 1 }, 'Only one filter icon is active after changing second metric');
+
+    await click(findAll('.navi-column-config-base__filter-icon')[1]);
+
+    assert
+      .dom('.navi-column-config-base__filter-icon--active')
+      .exists({ count: 2 }, 'Both filter icons are active after adding second parameterized metric filter');
+
+    assert.deepEqual(
+      findAll('.filter-builder__subject').map(el => el.textContent.trim()),
+      ['Date Time (Day)', 'Platform Revenue (CAD)', 'Platform Revenue (USD)'],
+      'Second metric filter is added'
+    );
+  });
+
+  test('config - filters - metrics - stay collapsed on remove', async function(assert) {
+    assert.expect(4);
+    await visit('/reports/new');
+    await click('.navi-report__run-btn');
+
+    await clickItem('metric', 'Ad Clicks');
+    await animationsSettled();
+
+    assert
+      .dom('.report-builder__container--filters')
+      .doesNotHaveClass('report-builder__container--filters--collapsed', 'Filters are open by default');
+    await click('.report-builder__container-header__filters-toggle-icon');
+
+    await click('.navi-column-config-item__name[title="Ad Clicks"]');
+    await click('.navi-column-config-base__filter-icon');
+
+    assert
+      .dom('.navi-column-config-base__filter-icon')
+      .hasClass('navi-column-config-base__filter-icon--active', 'The filter is active after being added');
+
+    await click('.report-builder__container-header__filters-toggle-icon');
+    assert
+      .dom('.report-builder__container--filters')
+      .hasClass('report-builder__container--filters--collapsed', 'Filters are collapsed after click');
+
+    await click('.navi-column-config-base__filter-icon');
+
+    assert
+      .dom('.report-builder__container--filters')
+      .hasClass('report-builder__container--filters--collapsed', 'Filters stay collapsed when a filter is removed');
+  });
+
+  test('config - filters - dimensions - stay collapsed on remove', async function(assert) {
```

Comment:
Do we need this test for parameterized metrics as well since it as an `expand on add` test?

Reply:
Added